shared/src/main/scala/mlscript/codegen/Scope.scala

```diff
@@ -51,16 +51,6 @@ class Scope(name: Str, enclosing: Opt[Scope]) {
     }
   }
 
-  /**
-    * Shorthands for creating function scopes.
-    */
-  def this(name: Str, params: Ls[Str], enclosing: Scope) = {
-    this(name, Opt(enclosing))
-    params foreach { param =>
-      declareValue(param)
-    }
-  }
-
```

Comment:
See, this is exactly what I was telling you about being careful with merges... It is not enough to just click "accept both". Please be more mindful next time.

Reply:
👀 Sorry, I didn't get. I deleted this because of [this comment](https://github.com/hkust-taco/mlscript/pull/123#discussion_r927398129). I didn't click any “accept both” button we deleting this.

Reply:
The point is that this constructor was deleted on MLScript's main branch. It was deleted in https://github.com/hkust-taco/mlscript/pull/117, on which you rebased your changes. You reintroduced it because you botched your merge. Please be more careful.